src/components/pages/AdminHome/ApplicationCard.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Card } from 'antd';
+import { connect, useDispatch } from 'react-redux';
+import { getInstructors } from '../../../redux/actions/instructorActions';
+import { useOktaAuth } from '@okta/okta-react';
+import '../../../styles/AdminDashboardHome/index.less';
+
+const ApplicationCard = props => {
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const { authState } = useOktaAuth();
+
+  useEffect(() => {
+    if (!authState) return;
+    dispatch(getInstructors(authState.idToken.idToken));
```

Comment:
I believe this should be `useDispatch` not `dispatch`. I may be wrong. Let's try to resolve the warning shown below.

Reply:
just needed to add dispatch to the dependency array in the useEffect